code/modules/food_and_drinks/recipes/tablecraft/recipes_burger.dm

```diff
@@ -309,6 +309,17 @@
 	result = /obj/item/food/burger/empoweredburger
 	subcategory = CAT_BURGER
 
+/datum/crafting_recipe/food/catburger
+	name = "Catperson burger"
```

Comment:
technically this can be made with normal cats too.

Reply:
renamed it to cat burger